server/game/cards/07-GR/CurseOfFertility.js

```diff
@@ -0,0 +1,31 @@
+const Card = require('../../Card.js');
+
+class CurseOfFertility extends Card {
+    // Treachery. (This card enters play under your opponent’s control.)
+    //
+    // At the end of your turn, if you did not play a creature this
+    // turn, your opponent gains 1.
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            location: 'any',
+            targetController: 'any',
+            effect: [ability.effects.entersPlayUnderOpponentsControl()]
+        });
+
+        this.interrupt({
+            when: {
+                onRoundEnded: (_, context) =>
+                    context.player === this.game.activePlayer &&
+                    !!context.player.opponent &&
```

Comment:
The !! isn't really necessary in this context. It's only really needed if you need an actual boolean rather than truthiness. I'll merge it as is but just FYI